content/SDK/nodejs-sdk.mdx

```diff
@@ -59,37 +59,6 @@ import schemaContents from './get-started.cfgu.json';
 })();
 ```
 
-**With fs & path**:
```

Comment:
why did you remove it?

Reply:
its a legit way to create a schema instance

Reply:
I'm unsure why this was removed, I didn't intend to touch this file at all, this was done by `onPreInit`. I just ran it again and it undid this diff. 🤷🏻